### ISSUE TITLE
Not able to open card attachments

### ISSUE THREAD

**Issue by WAKIGA98** [nextcloud/deck#2627]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Describe the bug**
We attached files (txt and pdf) to cards and sometimes it is not possible to open the attachment.
After making an upload of pdf-Files the files are displayed in the cards details attachment as PDF-Files.
Leaving NextCloud and opening it some times later again the files may not be displayed as PDF-Files. Trying to open this files shows a JSON-File, e.g. see attached file "Attachment27.json.txt".
[Attachment27.json.txt](https://github.com/nextcloud/deck/files/5612104/Attachment27.json.txt)

**To Reproduce**
This is a random problem. Sometimes it happens, sometimes it works without any trouble.

I tried different file names:
- AttachmentTest1_working.pdf
- AttachmentTest2 Blank.pdf
- AttachmentTest3_UmlautÄ.pdf
- AttachmentTest4_Umlautö.pdf
- AttachmentTest5_SZß.pdf

but the problem is independent of the file name.

The problem is independent of the creator of the card / attacher of the file.
If the problem is "active" an upload of the same file shows it as "undefined content" and as pdf, see Screenshot

**Expected behavior**
Attached files shall be displayable everytime.

**Screenshots**
![UpdateFile](https://user-images.githubusercontent.com/74932616/100541340-a5c0b680-3243-11eb-9435-916e1c87ea71.png)

**Client details:**
Chromium 72.0.3626.121 (Offizieller Build) / Firefox 83.0 (32-Bit)
Raspbian 9.13 (32-Bit) /Windows
Device: Desktop

(I'm not the administrator of the installation!)
DECK       1.1.2
NextCloud 19.0.4

<details>
<summary>Server details</summary>

**Operating system**:

**Web server:**

**Database:**

**PHP version:**

**Nextcloud version:** (see Nextcloud admin page)

**Where did you install Nextcloud from:**

**Signing status:**

```
Login as admin user into your Nextcloud and access
http://example.com/index.php/settings/integrity/failed
paste the results here.
```

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```

**Nextcloud configuration:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or

Insert your config.php content here
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

</details>

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```

</details>

**Comment by WAKIGA98** [comment]
I've to confirm the bug is still existing after update to
DECK 1.2.2
NextCloud 19.0.4

**Comment by juliushaertl** [comment]
ANy errors in the browser console?

**Comment by WAKIGA98** [comment]
> 
> 
> ANy errors in the browser console?

Yes, also after update to DECK	1.2.7 on NextCloud	20.0.10 the topic is existing.

The browser console shows lots of messages starting with the login in NextCloud. The following message are displayed with opening a single DECK card:
![Console_DeckOpenCard](https://user-images.githubusercontent.com/74932616/129412597-608ea98d-f2e1-4fd0-a039-f38e75661369.png)
Trying to open an attachment without success the following messages are displayed
![Console_DeckOpenAttachment](https://user-images.githubusercontent.com/74932616/129412675-de587c8f-eb81-4879-b877-10889b5c2b33.png)

I'm not able to detect any rule to find get a hint on this bug. Some cards have attachments one possible to open and and two not.
